package.json

```diff
@@ -28,23 +28,28 @@
     "semi": false,
     "singleQuote": true,
     "trailingComma": "es5",
-    "plugins": ["@trivago/prettier-plugin-sort-imports"],
-    "importOrder": ["^[./]"],
+    "plugins": [
+      "@trivago/prettier-plugin-sort-imports"
+    ],
+    "importOrder": [
+      "^[./]"
+    ],
     "importOrderSortSpecifiers": true,
     "importOrderCaseInsensitive": true,
     "importOrderSeparation": true
   },
   "dependencies": {
     "debug": "^4.3.7",
     "make-error": "^1.3.6",
-    "prom-utils": "^0.14.0"
+    "prom-utils": "^0.16.0"
   },
   "devDependencies": {
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "@types/debug": "^4.1.12",
+    "@types/node": "^22.13.14",
```

Comment:
Incidental: I added `@types/node` because my linter was complaining about not being able to resolve `node:timers/promises`